src/hpx_init_params.cpp

```diff
@@ -0,0 +1,24 @@
+//  Copyright (c)      2020 ETH Zurich
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <hpx/hpx_init_params.hpp>
+#include <hpx/runtime/shutdown_function.hpp>
+#include <hpx/runtime/startup_function.hpp>
+#include <hpx/type_support/unused.hpp>
+
+
+namespace hpx { namespace detail {
+
+    const hpx::program_options::options_description default_desc(
+                "Usage: " HPX_APPLICATION_STRING " [options]");
```

Comment:
This can't be in a `.cpp` file as the macro `HPX_APPLICATION_STRING` is define specifically for each executable.

Reply:
oh yeah sorry !